swift/Tests/Blockchains/NULSTests.swift

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import TrustWalletCore
+import XCTest
+
+class NULSTests: XCTestCase {
+
+    func testAddress() {
+        let key = PrivateKey(data: Data(hexString: "a1269039e4ffdf43687852d7247a295f0b5bc55e6dda031cffaa3295ca0a9d7a")!)!
+        let pubkey = key.getPublicKeyEd25519()
+        let address = NULSAddress(publicKey: pubkey)
+        let addressFromString = NULSAddress(string: "NULSd6HghWa4CN5qdxqMwYVikQxRZyj57Jn4L")!
+
+        XCTAssertEqual(address.description, addressFromString.description)
+    }
+
+    func testSign() {
+        let input = NULSSigningInput.with {
+            $0.private_key = "9ce21dad67e0f0af2599b41b515a7f7018059418bab892a7b68f283d489abc4b"
+            $0.from_address = "NULSd6Hgj7ZoVgsPN9ybB4C1N2TbvkgLc8Z9H"
```

Comment:
can you rename `from_address` to `from`, and same for `to`

Reply:
Renaming will change many code. Is it mandatory?

Reply:
I change from_address to from and to_address to to.